manifests/repo/debian.pp

```diff
@@ -24,16 +24,16 @@
 #   Enable special sury handling
 #
 class php::repo::debian(
-  $location     = 'http://packages.dotdeb.org',
-  $release      = 'wheezy-php56',
-  $repos        = 'all',
-  $include_src  = false,
-  $key          = {
+  String           $location     = 'http://packages.dotdeb.org',
```

Comment:
if we use the auto stuff i described below, we also should set the default location and key hash to undef